a_sample_module_template/a_sample_module_template.md

```diff
@@ -20,7 +20,9 @@ After completion of this module, learners will be able to:
 
 @end
 
-link:  https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/modules.css
+link:  https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/styles_icons.css
```

Comment:
Should we rename the file in AWS (or the one tracked here) so that they are the same?

Reply:
Yes, good call. I changed it here to styles.css since that's what we have in the repo, and will update on AWS.